tests/pkcli/build_data/1.out/srv/host/v9.radia.run/000.sh

```diff
@@ -26,6 +26,7 @@ rsconf_require dovecot
 rsconf_require github_bkp
 rsconf_require jupyterhub
 rsconf_require jupyterhub_proxy
+rsconf_require mlflow
```

Comment:
This file isn't necessary to have in the saved out.

Reply:
d599c0c